products-service/catalogBatchProcess.js

```diff
@@ -0,0 +1,62 @@
+import { DynamoDB } from 'aws-sdk';
+import { SNSClient, PublishCommand } from "@aws-sdk/client-sns";
+
+const dynamodb = new DynamoDB.DocumentClient({ region: "eu-west-1"});
+const topicArn = "arn:aws:sns:eu-west-1:654654363101:new-products-topic";
```

Comment:
same here, hardcoded values is not flexible and it may change during infra changes